setup.py

```diff
@@ -26,8 +26,8 @@ def read(*rnames):
         return f.read()
 
 setup(name='zope.size',
-      version='4.2.0.dev0',
-      url='http://pypi.python.org/pypi/zope.size',
+      version=read('version.txt'),
+      url='http://zopesize.readthedocs.io',
```

Comment:
Previously we (mostly me) changed the `url` to point to github but pointing to the documentation seems to be the better way.